edvart/report_sections/group_analysis.py

```diff
@@ -548,26 +81,23 @@ def required_imports(self) -> List[str]:
             List of import strings to be added at the top of the generated notebook,
             e.g. ["import pandas as pd", "import numpy as np"]
         """
-        ga = "GroupAnalysis"  # pylint:disable=invalid-name
         if self.verbosity == Verbosity.LOW:
-            return [
-                f"from edvart.report_sections.group_analysis import {ga}\n"
-                f"group_analysis = {ga}.group_analysis"
-            ]
+            return ["from edvart.report_sections.group_analysis import show_group_analysis"]
         if self.verbosity == Verbosity.MEDIUM:
             return [
-                total_dedent(
-                    f"""
-                        from edvart.report_sections.group_analysis import {ga}
-                        group_missing_values = {ga}.group_missing_values
-                        within_group_stats = {ga}.within_group_stats
-                        within_group_descriptive_stats = {ga}.within_group_descriptive_stats
-                        within_group_quantile_stats = {ga}.within_group_quantile_stats
-                        group_barplot = {ga}.group_barplot
-                        overlaid_histograms = {ga}.overlaid_histograms
-                        default_group_quantile_stats = {ga}.default_group_quantile_stats
-                        default_group_descriptive_stats = {ga}.default_group_descriptive_stats
+                code_dedent(
                     """
+                    from edvart.report_sections.group_analysis import (
```

Comment:
nitpick: these two lines should be joined when `total_dedent` is not used anymore.

Reply:
Done (same thing in univariate analysis)